charts/stable/zalando-postgres-cluster/values.yaml

```diff
@@ -59,8 +59,12 @@ persistentVolumes:
 
 
 dumpBackup:
-  # Enable backups to a PVC
+  # -- Enable backups to a PVC
+  enabled: false
+  # -- exiting claim
   existingClaim:
+  # -- type of persistence volume to use
```

Comment:
Maybe add some guidance about the options here, or refer to the storage docs?

Reply:
Added